docs/linting/troubleshooting/Formatting.mdx

```diff
@@ -55,6 +55,18 @@ module.exports = {
 
 Note that even if you use a formatter other than `prettier`, you can use `eslint-config-prettier` as it exclusively turns **off** all formatting rules.
 
+#### `eslint-plugin-prettier`
+
+`eslint-config-prettier` is not the same as [`eslint-plugin-prettier`](https://github.com/prettier/eslint-plugin-prettier).
+
+- The _config_ only disables rules from core and other plugins.
+- The _plugin_ loads and runs Prettier inside ESLint.
+
+We don't recommend using `eslint-plugin-prettier`:
```

Comment:
Heh, it's an interesting bit of nuance... I think we're solidly in our right to recommend against `eslint-plugin-prettier` on the _performance_ grounds given how we significantly change ESLint performance. But I can see the logic in not taking a stance on the _usability_ within ESLint core's area of concerns.

Reply:
Added a link to the [Prettier docs](https://prettier.io/docs/en/cli#--check), and noticed that they recommend `--check` for a human readable error in CI instead of `--list-different`. Switched us and the docs to use `--check` instead. Does this seem better?